.github/actions/setup-and-cache-rust/action.yml

```diff
@@ -12,6 +12,9 @@ inputs:
     description: Additional components to install that don't come with the toolchain by default.
     required: false
     default: ''
+  cache-key-prefix:
+    description: A custom cache key prefix to be used instead of the rust target. Must be used if there are multiple rust targets because keys cannot contain commas.
```

Comment:
Nit: I think splitting this line would look nicer -- no reason to have ~160 character lines.